backend/tests/v5/unit/processors/teamspaces/projects/models/containers.test.js

```diff
@@ -205,9 +220,34 @@ const testGetContainerStats = () => {
 	});
 };
 
+const testGetRevisions = () => {
+	describe('Get container revisions', () => {
+		test('should return non-void revisions if the container exists', async () => {
+			const res = await Containers.getRevisions('teamspace', 1, false);
+			expect(res).toEqual(model1Revisions.filter((r) => !r.void));
```

Comment:
you're testing your mock implementation here, not the source code.

It's probably better to just mock Revisions.getRevisions and check it's being called with the right parameters.